portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -64,17 +65,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get input from the form for comment text.
     String text = CommonUtils.getParameter(request, TEXT, /* DefaultValue= */ "");
 
+    // Get text with the sentiment tag added.
+    String textWithSentiment = null;
```

Comment:
Could you avoid null here?

http://www.javapractices.com/topic/TopicAction.do?Id=134

Reply:
Done - I think it makes sense to set it to text.